config/main.py

```diff
@@ -85,7 +85,7 @@ def set_interface_mode(mode):
     if user != "root":
         bashrc = "/home/{}/.bashrc".format(user)
     else:
-        raise click.Abort()
+        click_context.fail("Error: Cannot set interface naming mode for root user!")
```

Comment:
>click_context [](start = 8, length = 13)

[https://click.palletsprojects.com/en/7.x/advanced/#global-context-access](https://click.palletsprojects.com/en/7.x/advanced/#global-context-access)
may help reduce click_context. so the change will be minimized.